doc/reference/power_management/index.rst

```diff
@@ -29,11 +29,12 @@ Terminology
    one has a different meaning.
 
 :dfn:`Device Runtime Power Management`
-   Device Runtime Power Management (PM) refers the capability of
-   devices be able of saving energy independently of the the
-   system. Devices will keep reference of their usage and will
-   automatically be suspended or resumed. This feature is enabled via
-   the ::option:`CONFIG_PM_DEVICE_RUNTIME` Kconfig option.
+   Device Runtime Power Management (PM) refers to the capability of
+   devices to save energy independently of the system.
+   When enabled, devices keep reference counts of their usage and will
+   automatically be suspended or resumed when not needed.
```

Comment:
this `or resumed when not needed` sounds wrong, maybe
"and resumed when needed."